ecosystem/payara-maven/pom.xml

```diff
@@ -67,90 +67,184 @@
                     </execution>
                 </executions>
             </plugin>
-      
-            <plugin>
-                <groupId>org.codehaus.cargo</groupId>
-                <artifactId>cargo-maven2-plugin</artifactId>
-                <version>1.7.16</version>
-                <configuration>
-                    <skip>${skipTests}</skip>
-                </configuration>
-                <executions>
-                    <execution>
-                        <id>deploy</id>
-                        <phase>pre-integration-test</phase>
-                        <goals>
-                            <goal>redeploy</goal>
-                        </goals>
-                    </execution>
-                    <execution>
-                        <id>undeploy</id>
-                        <phase>post-integration-test</phase>
-                        <goals>
-                            <goal>undeploy</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
         </plugins>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.codehaus.cargo</groupId>
```

Comment:
We have our custom version of the Cargo plugin to handle the Deployment group (See FISH-87 and FISH-438)  Does that also work, What is the difference with this 1.9.1 version?

Reply:
I've just teested it with our custom Cargo plugin and redeploy and undeploy don't work in remote mode. All works fine in the local mode and loca-managed mode.